package.json

```diff
@@ -29,6 +29,7 @@
     "asset-pipe-common": "^1.0.0-beta.6",
     "boom": "^7.1.1",
     "browserify": "^15.2.0",
+    "envify": "^4.1.0",
```

Comment:
still not `loose-envify`? should be quicker, and the build is already pretty slow

Reply:
This worries me a little though:

> Doesn't replace oddly-spaced or oddly-commented expressions.
this won't work:
> `console.log(process./*won't*/env./*work*/NODE_ENV);`

I'm guessing 

```js
process
  .env
  .NODE_ENV
```
would also break it. Unlikely to happen much if at all but if it did it be a pita.

Reply:
I'm just afraid that this will become too slow. We are already so slow that our users are affected (missing js/css right after a deploy)